docs/source/LinearRegression.py

```diff
@@ -170,3 +171,43 @@ def g(X):
 plt.grid(True, ls="--", zorder=0, alpha=0.33)
 plt.legend()
 plt.show()
+
+# %%
+import scipy.sparse as sp
+from graphite_maps.linear_regression import linear_l1_regression
+
+# %%
+X_prior = np.copy(X)
+Y = g(X_prior)
+D = Y + rng.standard_normal(size=Y.shape)
```

Comment:
This line should use `covariance` or `obs_sd`. It works now, but only because `obs_sd=1.0`.